README.md

```diff
@@ -0,0 +1,151 @@
+# sanctuary-identity
+
+<img alt="Fantasy Land" src="https://raw.githubusercontent.com/fantasyland/fantasy-land/master/logo.png" width="200" height="200" align="right">
+
+Identity is the simplest container type: a value of type `Identity a`
+always contains exactly one value, of type `a`.
+
+`Identity a` satisfies the following [Fantasy Land][] specifications:
+
+  - [Setoid][] (if `a` satisfies Setoid)
+  - [Ord][] (if `a` satisfies Ord)
+  - [Semigroup][] (if `a` satisfies Semigroup)
+  - [Functor][]
+  - [Apply][]
+  - [Applicative][]
+  - [Chain][]
+  - [Monad][]
+  - [Alt][] (if `a` satisfies Alt)
```

Comment:
Thank you for the review, @syaiful6. I actually removed this line from __index.js__ in 7a64a8317d6a8deb62aa956cab7ab4988645adb0, but forgot to update the readme. I'm sorry about that. I'll remove the readme from this pull request to avoid further confusion.

Reply:
I agree. There is `instance (Alternative f) => Alternative (IdentityT f)`, but that only works because `IdentityT` is `* -> * -> *`. `IdentityT List String` is distinct from `Identity (List String)`.